spec/cypress/support/axe.js

```diff
@@ -6,7 +6,7 @@ Cypress.Commands.overwrite("injectAxe", () => {
   // https://github.com/component-driven/cypress-axe/issues/6#issuecomment-726986454
   cy.window({ log: false }).then((win) => {
     // eslint-disable-next-line global-require
-    const axe = require("axe-core/axe.js");
+    const axe = require("axe-core/axe");
```

Comment:
Why do we need this change?

Reply:
JS lint does not like extensions being included in `require`. I have no clue why it was complaining about this on this PR...